blogs/2024/11/11/introducing-github-copilot-for-azure.md

```diff
@@ -0,0 +1,78 @@
+
```

Comment:
---
Order: 89
TOCTitle: GitHub Copilot for Azure
PageTitle: Introducing GitHub Copilot for Azure
MetaDescription: Introducing GitHub Copilot for Azure, a chat participant to ask about Azure and help you manage and troubleshoot your Azure resources.
Date: 2024-11-12
Author: Chris Harris
---

Reply:
Done